<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ChangeMaskDialog</class>
 <widget class="QDialog" name="ChangeMaskDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import options</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>180</y>
     <width>162</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>180</y>
     <width>158</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <layout class="QHBoxLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
   </layout>
   <widget class="QSplitter" name="splitter_2" >
    <property name="orientation" >
     <enum>Qt::Horizontal</enum>
    </property>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>8</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
    </layout>
    <widget class="QRadioButton" name="AllButton" >
     <property name="text" >
      <string>All</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="NoneButton" >
     <property name="text" >
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="check_iom_camera" >
    <property name="text" >
     <string>Camera</string>
    </property>
    <property name="checkable" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title" >
    <string>Vert</string>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QCheckBox" name="check_iom_vertflags" >
      <property name="text" >
       <string>Flags</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_vertcolor" >
      <property name="text" >
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_vertquality" >
      <property name="text" >
       <string>Quality</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_vertnormal" >
      <property name="text" >
       <string>Normal</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_verttexcoord" >
      <property name="text" >
       <string>TextCoord</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>20</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Wedg</string>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QCheckBox" name="check_iom_wedgcolor" >
      <property name="text" >
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_wedgtexcoord" >
      <property name="text" >
       <string>TexCoord</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_wedgnormal" >
      <property name="text" >
       <string>Normal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>20</y>
     <width>111</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title" >
    <string>Face</string>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>8</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QCheckBox" name="check_iom_faceflags" >
      <property name="text" >
       <string>Flags</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_facecolor" >
      <property name="text" >
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_facequality" >
      <property name="text" >
       <string>Quality</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_iom_facenormal" >
      <property name="text" >
       <string>Normal</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>check_iom_vertflags</tabstop>
  <tabstop>check_iom_vertcolor</tabstop>
  <tabstop>check_iom_vertquality</tabstop>
  <tabstop>check_iom_vertnormal</tabstop>
  <tabstop>check_iom_verttexcoord</tabstop>
  <tabstop>check_iom_faceflags</tabstop>
  <tabstop>check_iom_facecolor</tabstop>
  <tabstop>check_iom_facequality</tabstop>
  <tabstop>check_iom_facenormal</tabstop>
  <tabstop>check_iom_wedgcolor</tabstop>
  <tabstop>check_iom_wedgtexcoord</tabstop>
  <tabstop>check_iom_wedgnormal</tabstop>
  <tabstop>AllButton</tabstop>
  <tabstop>NoneButton</tabstop>
  <tabstop>check_iom_camera</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeMaskDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ChangeMaskDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
